### ISSUE TITLE
Cfar10 Accuracy seems very low.

### ISSUE THREAD

**Issue by fabacha** [Guncuke/GraduationProject#1]
Hi!

Thank you for making your implementation open source.

Do you have an idea why the CIFAR 10 accuracy is very low?

Thank you

**Comment by Guncuke** [comment]
> Hi!
> 
> Thank you for making your implementation open source.
> 
> Do you have an idea why the CIFAR 10 accuracy is very low?
> 
> Thank you

You can experiment with a trained cifar-10 model and load the trained model in models.py

If you are talking about the backdoor accuracy is not high, I also found that the effect of pixel attacks on RGB graphs does not seem to be very obvious, by the way, the magnification attack in clients.py was commented out

**Comment by fabacha** [comment]
Thank you for the prompt response.

I am new to ML so I was referring to the accuracy of the model.

I will try using the pretrained model. Can you please recommend any pretrained model I can use?

Thank you

**Comment by Guncuke** [comment]
> 
I recommend that you train your own model and use resnet18 to train a neural network to recognize cifar-10. resnet18 is not very deep so it is fast to train.

**Comment by fabacha** [comment]
Okay.
Thank you.
